usaspending_api/download/helpers/download_annotation_functions.py

```diff
@@ -139,6 +139,63 @@ def _outlay_amount_agg_subquery(
     )
 
 
+def _outlay_amount_agg_subquery_no_coalesce(
+    emergency_fund_group_name: str = None,
+    def_codes: Optional[List[str]] = None,
+    award_id_col: Optional[str] = "award_id",
+) -> Subquery:
+    """This function returns a total outlay aggregation
+    where NULLs are not represented as 0s. This function
+    is useful for when you want to show data that doesn't have outlay values
+    as a blank/NULL (i.e. do NOT show “0”).
+    """
+    filters = [filter_by_latest_closed_periods(), Q(award_id=OuterRef(award_id_col))]
+    if emergency_fund_group_name is not None:
+        filters.append(Q(disaster_emergency_fund__group_name=emergency_fund_group_name))
+    if def_codes:
+        filters.append(Q(disaster_emergency_fund__code__in=def_codes))
+
+    return Subquery(
+        FinancialAccountsByAwards.objects.filter(*filters)
+        .values("award_id")
+        .annotate(
+            total_gross_outlay_amount_by_award_cpe=Sum("gross_outlay_amount_by_award_cpe"),
```

Comment:
How does Django handle `NULL` values in these `Sum()` aggregations?

Reply:
NULLs are ignored.